core/src/main/java/org/apache/druid/java/util/common/guava/ParallelMergeCombiningSequence.java

```diff
@@ -1376,6 +1382,6 @@ long getTotalCpuTimeNanos()
   {
     Closer closer = Closer.create();
     closer.registerAll(cursors);
-    CloseQuietly.close(closer);
+    CloseableUtils.closeAndWrapExceptions(closer);
```

Comment:
Looking at the usage of this method, it appears that it is expected to eat the exceptions:

```
      catch (Throwable t) {
        closeAllCursors(sequenceCursors);
        cancellationGizmo.cancel(t);
        out.offer(ResultBatch.TERMINAL);
      }
```
so, I think this method should be using `closeAndSuppressExceptions`. I guess callers could also be re-arranged to do the things that absolutely need done before closing the cursors, however, nothing would directly catch an exception thrown here, so it is sort of like screaming into the void, so I think we should just suppress them.

Reply:
I'll change it to `CloseableUtils.closeAndSuppressExceptions`